src/errors.rs

```diff
@@ -27,6 +31,7 @@ impl fmt::Display for DaachorseError {
     }
 }
 
+#[cfg(feature = "std")]
 impl Error for DaachorseError {}
```

Comment:
It seems that `DaachorseError` implementations for `std` features in `errors` module are not needed.
Can we remove them?

Reply:
Errors are often propagated as `Box<dyn std::errors::Error>`, so I think this is necessary.

Reply:
I see.